src/dom/set.js

```diff
@@ -11,9 +11,11 @@ function set (subject, property, value) {
 	else if (property in subject) {
 		subject[property] = value;
 	}
-	else {
+	else if (property) {
```

Comment:
Why remove?

Reply:
I'm sorry, I didn't get the question. Why remove what? I suggest adding `if (property)` to avoid cases like that:
`$.create()` which otherwise will give us `<div undefined></div>`. 

What am I missing?

Reply:
Ah, i misread the diff, sorry (trying to do too many things at once)